src/react-native/07-debugging-devtools/debugging-devtools.md

```diff
@@ -92,6 +95,16 @@ This will open up the dev menu. It should look like the following image:
 
 Clicking on `Open Debugger` will open up a separate window for debugging.
 
+### Exercise 2
+
+- Add `console.log` Statements:
+  - Within the `src/screens/StateList/StateList.tsx`, insert console.log statements to print information such as variable values, function calls, or component lifecycle events.
```

Comment:
I think these should actually be a part of the Setup, so during Setup we would have them add the `debugger` script and add the `console` logs and `debugger` statement, then… maybe there’s nothing to Verify, and then the Exercise is to launch the debugger and see what they get.

If we tell them exactly where to put `console.info` and `debugger` code, then our screenshot can show them exactly what they should see in their DevTools inspector.